app/models/concerns/thredded/post_common.rb

```diff
@@ -81,7 +81,7 @@ def previous_post
     protected
 
     def update_unread_posts_count
-      postable.user_read_states.update_post_counts!
+      postable&.user_read_states&.update_post_counts!
```

Comment:
if the postable exists then so will `user_read_states`
Instead, let's add `, if: :postable` on line 43